common/dtos.go

```diff
@@ -52,3 +52,12 @@ type InitialAccountAPI struct {
 	StakingValue string            `json:"stakingvalue"`
 	Delegation   DelegationDataAPI `json:"delegation"`
 }
+
+// TODO: extend this structure to include more fields
```

Comment:
I think we can safely remove the TODO.

Reply:
as discussed, in order to simplify the logic, I wouldn't compute the hash, since I need the structure, not the interface